config/perm/events.js

```diff
@@ -9,6 +9,7 @@ module.exports = function () {
       	cmd: 'is_ticketing_admin'
       }]
     }],
+    //  TODO: verify this one, seems meh to me
```

Comment:
If events doesn't returns the invited people by default, i'd say yes, didn't fully verified returned value @butlerx  ?